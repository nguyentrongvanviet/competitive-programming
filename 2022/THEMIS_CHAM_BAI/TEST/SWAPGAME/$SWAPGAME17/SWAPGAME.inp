101
96 93 87 84 25 60 14 7 43 89 45 6 29 51 3 7 98 27 36 47 66 28 19 51 59 68 67 62 44 41 63 76 14 91 18 55 33 34 8 75 63 54 12 70 62 24 57 94 70 64 37 56 23 64 82 89 61 32 77 5 44 22 97 35 95 23 77 53 86 46 74 50 66 92 30 80 101 26 39 9 15 12 83 40 10 78 99 19 35 20 82 69 47 16 71 9 38 72 28 65 73 
99 56 38 13 83 84 49 49 58 67 45 58 1 91 88 37 75 46 93 31 6 80 31 13 2 10 54 4 97 74 79 42 34 90 18 61 48 88 3 98 22 65 55 92 21 24 85 15 17 79 8 36 81 101 100 69 5 81 43 30 11 57 11 59 32 95 68 100 29 50 20 72 4 85 60 86 90 53 87 48 94 17 25 52 21 73 96 40 2 41 26 76 52 78 39 33 71 27 1 42 16 