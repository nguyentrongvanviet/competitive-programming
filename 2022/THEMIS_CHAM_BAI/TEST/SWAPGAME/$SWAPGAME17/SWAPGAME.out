1048576
25